background.js

```diff
@@ -0,0 +1,6 @@
+chrome.runtime.onInstalled.addListener(function() {
+        const interval_seconds = 30
+	chrome.storage.sync.set({refresh_interval: interval_seconds}, function() {
+		console.log(`The refresh interval is ${interval_seconds} seconds`);
```

Comment:
```suggestion
        const refreshInterval = 30
	chrome.storage.sync.set({refreshInterval: refreshInterval}, function() {
		console.log(`The refresh interval is ${refreshInterval} seconds`);
```

Too much Python has me defaulting to snake_case these days.